### DISCUSSION TITLE
Zigbee devices backup

### DISCUSSION THREAD

**Discussion by marjankow** [arendst/Tasmota#22069] [Q&A]
Is it possible to backup zigbee devices and load them later on the same gateway (after reset for example). Or load them on some other gateway? It is really hard to pair everything again.

**Comment by s-hadinger** [comment]
No and yes...

The actual zigbee pairing in managed by the MCU, which has its own closed format. You can't backup them.

However, you can transfer the Tasmota mapping (friendly name, last know values...) which are stored in `zbv4` and `zbdata` files. It will save you some configuration time when you pair on the new coordinator

**Reply by marjankow** [reply]
How to download zbdata? I was able to download zbv4 from Manage File system

**Reply by s-hadinger** [reply]
`zbdata` is in the file system as well. If it is not present then it's content is empty and you don't need to care

**Reply by marjankow** [reply]
When I execute zbdata command via console I see some data 
stat/zigbee-bridge-garage/RESULT = {"ZbData":"ZbData 0x77AD,0B2F1ED6662A5139E7D46600040E0101000506010F0000"}
and so on for other devices. When I replace zbv4 on other gateway with the zb4 from above gateway nothing has been changed

**Reply by sfromis** [reply]
You get nothing from using those files, unless you already have paired the end device.
